code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
+'''
+
+import numpy as np
+import os
+from collections import Counter
+
+class NlpData:
+
+    data = None
+    data_only_ids = None
+    vocab = None
+    current_batch = 0
+
+    def get_next_batch(self, batch_size):
```

Comment:
@StefanGStark Just keep in mind to handle the last batch. I fixed it by replicating the last line to fill the batch (this was asked in the Q&A and the TAs gave us this hint). If you have a better solution, even better :)